test-kit/components/selection-list-driver.ts

```diff
@@ -1,11 +1,16 @@
 import {DriverBase, simulate} from 'test-drive-react';
 import {SelectionList} from '../../src';
+import optionBaseStyle from '../../src/components/selection-list/option.st.css';
 import listBaseStyle from '../../src/components/selection-list/selection-list.st.css';
 import {elementHasStylableClassName, elementHasStylableState} from '../utils';
 
 export class SelectionListTestDriver extends DriverBase {
     public static ComponentClass = SelectionList;
 
+    public get divider(): Element {
```

Comment:
We shouldn't do this, since the list can contain multiple dividers. What was the problem with the test? Maybe we could solve it differently, for example by defining `isDivider()` in the test driver?

Reply:
okay, I will think what to do here, this made because this test
` expect(list.elementHasStylableClassName(list.items[0], 'divider')).to.equal(true)` was too specific and broken on making divider's as seaprate component.

Reply:
@alisey so thanks, I've added style argument to `elementHasStylable*` methods.